src/test/java/io/confluent/connect/elasticsearch/ElasticsearchClientTest.java

```diff
@@ -331,7 +330,17 @@ public void testDeleteRecord() throws Exception {
     waitUntilRecordsInES(2);
 
     // delete 1
-    SinkRecord deleteRecord = new SinkRecord(TOPIC, 0, Schema.STRING_SCHEMA, "key0", null, null, 3);
+    SinkRecord deleteRecord = new SinkRecord(
```

Comment:
would be nice to abstract all of these to a helper method since they take a lot of space